packages/valory/contracts/gnosis_safe/tests/test_contract.py

```diff
@@ -52,9 +53,18 @@
     GnosisSafeContract,
     SAFE_CONTRACT,
 )
+from packages.valory.contracts.gnosis_safe_proxy_factory.tests.test_contract import (
+    PACKAGE_DIR as PROXY_DIR,
+)
+
 
-from tests.conftest import ROOT_DIR, THIRD_PARTY_CONTRACTS
+PACKAGE_DIR = Path(__file__).parent.parent
+THIRD_PARTY_CONTRACTS = Path(
+    os.environ.get("THIRD_PARTY_CONTRACTS", PACKAGE_DIR / "third_party")
+)
```

Comment:
We might wanna introduce registry images soon because this will just get harder to maintain over the time.

Reply:
This is messy indeed. I already started creating the registry image, so it should be easy to add next week! Please open an issue so we don't forget about the removal.